pkg/kubelet/kubelet.go

```diff
@@ -496,10 +490,19 @@ func NewMainKubelet(
 		containerRefManager,
 		recorder)
 
-	if err := klet.volumePluginMgr.InitPlugins(volumePlugins, &volumeHost{klet}); err != nil {
+	klet.volumePluginMgr, err =
+		NewInitalizedKubeletVolumePluginMgr(klet, volumePlugins)
```

Comment:
It calls `InitPlugins` on the newly constructed manager before returning it so that caller doesn't have to.